modules/gnome-extensions/README.md

```diff
@@ -55,3 +55,27 @@ How to uninstall extensions using the module:
 1. Go to Gnome Extensions app, https://extensions.gnome.org/local/ or [Extension Manager](https://github.com/mjakeman/extension-manager) application
 2. List of installed system extensions should be presented
 3. Copy the extension name & input it in module recipe (it is case-sensitive, so be sure that you copied it correctly)
+
+## Known issues
+
+### Multiple compatible Gnome extensions with the same name are not supported
```

Comment:
One would probably still want these sorts of extensions installed, so wouldn't it make sense to recommend one to use the legacy method here? That's what I opted for with Panel Date Format. I believe the legacy method would need to be redocumented, though.

Reply:
Also, one module definition currently cant mix legacy & new config types, but that does make it a bit more clumsy to use both.

Reply:
I'll think about refactoring the code to additionally support UUID in some way, rather than advising users to use legacy config.

Reply:
@xynydev I find out how to support specifying UUID:

We should use `extension-info` instead of `extension-query`
Thanks to looking at Extension Manager source code.

Example of newer `Panel Date Format:`  
https://extensions.gnome.org/extension-info/?pk=3465

Example of old `Panel Date Format`:  
https://extensions.gnome.org/extension-info/?pk=1462

I'll try to think on how to handle the best, transition to new UUID naming again, since it's preferred.

Reply:
I'm also not sure If I should ditch this PR & start working on new one, or merge this, as band-aid workaround.

Reply:
I think this PR is alright. We could support both specifying extensions with the names and with the pk/id. The error messages from this PR could then be amended to tell the user to use the extension id instead.